contrib/pg_prewarm/pg_prewarm.control

```diff
@@ -3,3 +3,4 @@ comment = 'prewarm relation data'
 default_version = '1.2'
 module_pathname = '$libdir/pg_prewarm'
 relocatable = true
+trusted = true
```

Comment:
I didn't find such researches. pg_prewarm just loads information in shared buffers or file system cache.
It may affect server performance, but can not somehow cause data leaks.
Running prewarm in background mode requires:
1. Adding pg_prewarm to [shared_preload_libraries](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-SHARED-PRELOAD-LIBRARIES) in postgresql.conf
2. Creation of `autoprewarm.blocks` file
None of this actions is available for Neon users.
So I do not see something dangerous if user will call `autoprewarm_dump_now` or `autoprewarm_start_worker` functions.

Reply:
> No, you can also call autoprewarm_start_worker

Yes, and it seems to be rally useful feature, especially for us. Because it is ready solution for speed-up node startup/restart. Is it dangerous? I don't think so. Yes, it will requires some space for autoprewarm.blocks file. But if you cache is small enough (i.e. 256Mb now), then it is negligible.

I am not familiar with how access control is currently performed in Neon (is it possible to define several users/roles accessing the same database?). But if  it is possible, then it is possible to prohibit particular user to create exeption by revoking create permission.